src/core/tsi/test_creds/README

```diff
@@ -62,6 +62,19 @@ common name which is set to testclient2.
 $ openssl x509 -req -CA ca.pem -CAkey ca.key -CAcreateserial -in client2.csr \
   -out client2.pem -days 3650
 
+client3 is issued by CA:
```

Comment:
Let's name is something more clear that this one is a valid SPIFFE ID
client_with_spiffe or something like that.
That way in the future we don't have to read through the `cnf` file to see what the distinguishing feature of this set of credentials is

Reply:
Renamed.